backend/projects/tasks.py

```diff
@@ -295,6 +298,11 @@ def export_project_in_place(
             # We need to store the rating in integer format
             if field == "rating":
                 setattr(data_item, field, int(ta[field]))
+            elif field == "transcribed_json":
+                for idx in range(len(ta_transcribed_json)):
+                    ta_labels[idx]["text"] = [ta_transcribed_json[idx]]

```

Comment:
We should also store the timestamps of the marked region.

Reply:
Can you please also check once if this JSON format matches with that of the format mentioned in the word doc (including speaker_id, etc.)